src/lib/registration/ui/migration_repos_workflow.rb

```diff
@@ -193,18 +208,33 @@ def find_products
         end
 
         merge_registered_addons
+
         log.info "Products to migrate: #{products}"
 
         :next
       end
 
+      # ask the user about adding not installed addons to the current products
+      #
+      # @return [Array<Hash>] installed products and addons selected to be installed
       def merge_registered_addons
         # load the extensions to merge the registered but not installed extensions
         Addon.find_all(registration)
 
-        addons = Addon.registered_not_installed.map(&:to_h).map do |addon|
-          SwMgmt.remote_product(addon)
-        end
+        # TRANSLATORS: Popup question, merge this addon that are registered but not
+        # installed to the current migration products list.
+        # %s is an addon friendly name, e.g 'SUSE Enterprise Storage 2 x86_64'
+        msg = "The '%s' extension is registered but not installed.\n" \
+              "If you accept it will be added for be installed, in other case " \
+              "it will be unregistered at the end of the migration.\n\n" \
+              "Do you want to add it?"
+
+        addons =
+          Addon.registered_not_installed.each_with_object([]) do |addon, result|
+            if Yast::Popup.YesNoHeadline(addon.friendly_name, _(msg % addon.friendly_name))
```

Comment:
... and `_(msg) % addon.friendly_name` here.

*Never* modify the string *before* translating it otherwise gettext will not find the translation for it.

See more details here: https://github.com/yast/yast.github.io/blob/master/doc/localization.md#translating-constants